### ISSUE TITLE
Precompiling a pch file Exit status: 65

### ISSUE THREAD

**Issue by gcgxfqw** [fastlane/fastlane#11271]
### New Issue Checklist

- [ ] Updated fastlane to the latest version
- [ ] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [ ] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

▸ Running script '[CP] Check Pods Manifest.lock'
▸ Running script '[CP] Check Pods Manifest.lock'
▸ Running script '[CP] Check Pods Manifest.lock'
▸ Precompiling jyxb_app/JYXB-Prefix.pch
** ARCHIVE FAILED **
[17:32:40]: Exit status: 65

+---------------+-------------------------+
|            Build environment            |
+---------------+-------------------------+
| xcode_path    | /Applications/Xcode.app |
| gym_version   | 2.69.3                  |
| export_method | ad-hoc                  |
| sdk           | iPhoneOS11.1.sdk        |
+---------------+-------------------------+

[17:32:40]: ▸ Verify final result code for completed build operation
[17:32:40]: ▸ Build operation failed without specifying any errors. Individual build tasks may have failed for unknown reasons.
[17:32:40]: ▸ One possible cause is if there are too many (possibly zombie) processes; in this case, rebooting may fix the problem.
[17:32:40]: ▸ Some individual build task failures (up to 12) may be listed below.
[INSERT OUTPUT HERE]

### Environment

<details><summary>🚫 fastlane environment 🚫</summary>

### Stack

| Key                         | Value                                       |
| --------------------------- | ------------------------------------------- |
| OS                          | 10.12.6                                     |
| Ruby                        | 2.4.1                                       |
| Bundler?                    | false                                       |
| Git                         | git version 2.13.6 (Apple Git-96)           |
| Installation Source         | ~/.rvm/rubies/ruby-2.4.1/bin/fastlane       |
| Host                        | Mac OS X 10.12.6 (16G1114)                  |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.4.1/lib                |
| OpenSSL Version             | OpenSSL 1.1.0g  2 Nov 2017                  |
| Is contained                | false                                       |
| Is homebrew                 | false                                       |
| Is installed via Fabric.app | false                                       |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/ |
| Xcode Version               | 9.1                                         |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | zh_CN.UTF-8 | ✅ |
| LC_ALL   |             |   |
| LANGUAGE |             |   |

### fastlane files:

<details><summary>`./Fastfile`</summary>

```ruby
# Customize this file, documentation can be found here:
# https://docs.fastlane.tools/actions/
# All available actions: https://docs.fastlane.tools/actions
# can also be listed using the `fastlane actions` command

# Change the syntax highlighting to Ruby
# All lines starting with a # are ignored when running `fastlane`

# If you want to automatically update fastlane if a new version is available:
# update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "2.69.3"

# 定义打包平台
default_platform :ios

FASTLANE_XCODEBUILD_SETTINGS_TIMEOUT=120

# 任务脚本
platform :ios do
lane :ci do|options|
branch = options[:branch]
# 自动获取当前app版本号，并补充当前编译时间为后缀
build_no = get_version_number + '.' + Time.new.strftime("%m%d%H%M")

puts "Begin to run ci"

# 递增build number
increment_build_number(build_number: build_no)

# 开始打包
gym(
# 是否清空以前的编译信息 true：是
clean:true,
# 指定打包方式，Release 或者 Debug
configuration:"Debug",
# 指定打包所使用的输出方式，目前支持app-store, package, ad-hoc, enterprise, development
export_method:"ad-hoc",
# 指定输出文件夹
output_directory:"./fastlane/build",
)

# 使用fir-cli上传ipa
# 请将SDAutolayoutDemo改成你项目要打包的名称
# 请将-T后的数字更改为在fir申请的APIToken
sh "fir publish ./build/jyxb_app.ipa -T a8e0cb9c1e76afb19ee6504baadf8454"

end
end

# Learn more at https://docs.fastlane.tools/#metrics

```
</details>

<details><summary>`./Appfile`</summary>

```ruby
app_identifier "com.xiaoyuxiaoyu.jyxb" # The bundle identifier of your app
apple_id "zhangqian@xiaoyuxiaoyu.com" # Your Apple email address

team_id "XS7VE74UZS" # Developer Portal Team ID

# you can even provide different app identifiers, Apple IDs and team names per lane:
# More information: https://docs.fastlane.tools/advanced/#appfile

```
</details>

### fastlane gems

| Gem      | Version | Update-Status      |
| -------- | ------- | ------------------ |
| fastlane | 2.69.3  | 🚫 Update available |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                       | Version      |
| ------------------------- | ------------ |
| did_you_mean              | 1.1.0        |
| executable-hooks          | 1.3.2        |
| bundler-unload            | 1.0.2        |
| rubygems-bundler          | 1.4.4        |
| bundler                   | 1.16.0       |
| io-console                | 0.4.6        |
| slack-notifier            | 1.5.1        |
| CFPropertyList            | 2.3.6        |
| claide                    | 1.0.2        |
| colored2                  | 3.1.2        |
| nanaimo                   | 0.2.3        |
| xcodeproj                 | 1.5.4        |
| rouge                     | 2.0.7        |
| xcpretty                  | 0.2.8        |
| terminal-notifier         | 1.8.0        |
| unicode-display_width     | 1.3.0        |
| terminal-table            | 1.8.0        |
| plist                     | 3.4.0        |
| public_suffix             | 2.0.5        |
| addressable               | 2.5.2        |
| multipart-post            | 2.0.0        |
| word_wrap                 | 1.0.0        |
| tty-screen                | 0.6.3        |
| babosa                    | 1.0.2        |
| colored                   | 1.2          |
| highline                  | 1.7.10       |
| commander-fastlane        | 4.4.5        |
| excon                     | 0.60.0       |
| faraday                   | 0.13.1       |
| unf_ext                   | 0.0.7.4      |
| unf                       | 0.1.4        |
| domain_name               | 0.5.20170404 |
| http-cookie               | 1.0.3        |
| faraday-cookie_jar        | 0.0.6        |
| fastimage                 | 2.1.0        |
| gh_inspector              | 1.0.3        |
| json                      | 2.0.2        |
| mini_magick               | 4.5.1        |
| multi_json                | 1.12.2       |
| multi_xml                 | 0.6.0        |
| rubyzip                   | 1.2.1        |
| security                  | 0.1.3        |
| xcpretty-travis-formatter | 1.0.0        |
| dotenv                    | 2.2.1        |
| faraday_middleware        | 0.12.2       |
| uber                      | 0.1.0        |
| declarative               | 0.0.10       |
| declarative-option        | 0.1.0        |
| representable             | 3.0.4        |
| retriable                 | 3.1.1        |
| mime-types-data           | 3.2016.0521  |
| mime-types                | 3.1          |
| little-plugger            | 1.1.4        |
| logging                   | 2.2.2        |
| jwt                       | 2.1.0        |
| memoist                   | 0.16.0       |
| os                        | 0.9.6        |
| signet                    | 0.8.1        |
| googleauth                | 0.6.2        |
| httpclient                | 2.8.3        |
| google-api-client         | 0.13.6       |
| openssl                   | 2.0.3        |
</details>

*generated on:* **2017-12-19**
</details>
[INSERT OUTPUT HERE]

**Comment by KrauseFx** [comment]
Seems like this is a build error, and not a fastlane error?

**Comment by shiyj** [comment]
It's not fastlane's problem. I had a similar error when I updated to xcode 9.2 and macOS 10.13.2, and I solved it by downgrading my cocoapods from 1.3.1 to 1.2.1.
